virttest/qemu_vm.py

```diff
@@ -896,25 +896,35 @@ def add_memorys(devices, params):
 
             :param devices: VM devices container
             """
-            options = []
+            options, devs = [], []
+            params.setdefault("mem", "512")
             mem_params = params.object_params("mem")
-            if mem_params.get("maxmem"):
+            mem_params.setdefault("automem", "yes")
```

Comment:
@balamuruhans  have sent pull-request for migration related case to set  automem = "no",  details see https://github.com/autotest/tp-qemu/pull/933.
Thanks your comments.
Xu